index.js

```diff
@@ -54,34 +67,58 @@
 // // => "different"
 // console.log("%c----------", "color: red") 
 
+function sameSameButDifferent(num, maybeNum) {
+  let result;
+  if (num === maybeNum) {
+    result = "same same";
+  } else if (num == maybeNum) {
+    result = "same same (but different)";
+  } else {
+    result = "different";
+  }
+  return result;
+}
+
 
 
 // ***** Question 5 *****
 
 // *** Uncomment the lines below to test
 // console.log("%cQuestion 5", "color: red")
 
-// const student1 = { name: "Duane", grade: 88 }
+const student1 = { name: "Duane", grade: 88 }
 // updateGrade(student1, 92)
 // console.log(student1)  
 // // => { name: "Duane", grade: 92 }
 // console.log("%c----------", "color: red") 
 
+function updateGrade(student, grade) {
+  student.grade = grade;
+  return student;
+}
 
 
 // ***** Question 6 *****
 
 // *** Uncomment the lines below to test
 // console.log("%cQuestion 6", "color: red")
 
-// const users = [ 
-//   { 
-//     name: "Duane", phones: { cell: "555-123-4567", office: "555-456-7890" }
-//   },
-//   { 
-//     name: "Liza", phones: { cell: "555-234-5678", office: "555-567-1234" }
-//   }
-// ]
+const users = [ 
+  { 
+    name: "Duane", phones: { cell: "555-123-4567", office: "555-456-7890" }
+  },
+  { 
+    name: "Liza", phones: { cell: "555-234-5678", office: "555-567-1234" }
+  }
+]
+
+function printNameAndPhones(arr) {
+  arr.forEach(element => {
+    console.log(element.name);
+    console.log(`Cell: ${element.phones.cell}`);
+    console.log(`Office: ${element.phones.office}`);
+  });
```

Comment:
Arrow function! Love to see it. There are a few tricky things about arrow functions but I'll have a video to go over them later in the week.